pep/app/pods/components/search/preview/template.hbs

```diff
@@ -62,7 +62,25 @@
                         {{html-safe @result.abstract}}
                     </div>
                     <div>
-                        {{#if this.session.isAuthenticated}}
+                        {{#if @result.accessLimited}}
+                            <Alert
+                                @isShown={{this.session.isAuthenticated}}
+                                @type="warning"
+                            >
+                                {{html-safe
+                                    (or
```

Comment:
This seems odd to me. Usually an `or` would have 3 items. Might be better as an `if`

Reply:
not sure why an `{{or}}` would usually have 3 items? This is pretty common, its just returning the first truthy value, basically equivalent to:
```ts
return this.args.result.noAccessMessage || this.intl.t('search.preview.accessLimited')
//or
return this.args.result.noAccessMessage ?? this.intl.t('search.preview.accessLimited')
```
which we use alot